driver.cpp

```diff
@@ -482,6 +600,18 @@ namespace bitcoinfuzz
             this->TransactionEvalTarget(buffer);
         } else if (target == "bip32_master_keygen") {
             this->Bip32MasterKeygenTarget(buffer);
+        } else if (target == "private_to_public")
+        {
+            this->PrivateToPublicTarget(buffer);
+        } else if (target == "sign_compact")
+        {
+            this->SignCompactTarget(buffer);
+        } else if (target == "sign_der")
+        {
+            this->SignDerTarget(buffer);
+        } else if (target == "ecdh")
+        {
+            this->ECDHTarget(buffer);
```

Comment:
nit: let's follow the same pattern of else if

```suggestion
        } else if (target == "private_to_public") {
            this->PrivateToPublicTarget(buffer);
        } else if (target == "sign_compact") {
            this->SignCompactTarget(buffer);
        } else if (target == "sign_der") {
            this->SignDerTarget(buffer);
        } else if (target == "ecdh") {
            this->ECDHTarget(buffer);
```

Reply:
Was about to ask what C++ formatter you are using? I can see there are a lot of places in my code where the formatting got changed when I format the document with my formatter. In other places in my code, the formatting is a little different as well then usual.

Reply:
Yes, we haven't actually been using a formatter. We should establish one to make formatting easier and avoid having to do it manually. Since we have many languages in this project, it would probably be best to establish a formatter for each language. I will open an issue to discuss that further.